etna/lib/etna/clients/magma/workflows/add_model_from_json_workflow.rb

```diff
@@ -126,9 +126,11 @@ def update_magma_attributes
         end
 
         def add!
+          puts "Adding the new model to Magma."
```

Comment:
I've been thinking that at some point we may want to make it easier to inject the application logger into workflows.  But I don't have a solution right now and I appreciate the point of this change. TODO